[SAASINT-3312] DDS: ExtraHop: Crawler Integration v1.0.0 (#18928)
DataDog/integrations-core: extrahop/README.md

# ExtraHop

## Overview

[ExtraHop][1] applies machine learning techniques and rule-based monitoring to your wire data to identify unusual behaviors and potential risks to the security and performance of your network.

This integration ingests the following logs:

- Detection: Represents data of anomalous behavior identified by ExtraHop system.
- Investigation: Represents a collection of data related to a specific security investigation, including its status, assignment, and associated detections.

This integration seamlessly collects all the above listed logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into detections and investigations through the out-of-the-box dashboards.

## Setup

### Generate API credentials in ExtraHop

1. Log in to the ExtraHop platform.

> 2. Go to **System Settings** > **API Access** and click **Create Credentials** which is present at the bottom of the page under **Rest API Credentials** section.

Reviewer: ```suggestion
2. Go to **System Settings** > **API Access** and click **Create Credentials** which is at the bottom of the page under **Rest API Credentials** section.
```

Reviewer 2: @jhgilbert ,
Done 👍

> 3. On **System Settings** > **API Access** > **Rest API Credentials**, at the top right corner; Click **Create Credentials** and specify the settings of the new Client ID and Client Secret.

Reviewer: ```suggestion
3. Navigate to **System Settings** > **API Access** > **Rest API Credentials**. In the top right corner, click **Create Credentials** and specify the settings of the new Client ID and Client Secret.
```

Reviewer 2: @jhgilbert,
Done 👍
